kubernetes_cluster_autoscaler/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# CHANGELOG - Kubernetes Cluster Autoscaler
+
+<!-- towncrier release notes start -->
+
+## 1.0.0 / 2024-05-15 / Agent 7.55.0
```

Comment:
Hmm, did you add this by hand? Normally this doesn't get added until we release the integration. And the Agent version also gets added later, after the agent releases.

Reply:
I added it manually as the checks for the PR kept falling kept failing